content/articles/web-apps.md

```diff
@@ -0,0 +1,108 @@
+---
+author: "Emma Russell"
+date: 2023-09-12
+title: "RESIDE-IC Web Applications"
+tags:
+ - web
+---
+
+# Introduction
+
+This page describes some aspects to think about when considering a new RESIDE web application to present research:
+
+- why a research team might want a web application
+- whether a Shiny app or full web application would better meet their needs
+- the components of a full research web application
+- what we require from research in order to build an interface to it which a new web application can use
+- some different patterns of application we can provide
+
+# Why have a web application?
+..as opposed to making research code available for users to download and run themselves?
```

Comment:
This line feels very sassy :laughing: not a bad thing, just funny

Reply:
It was genuinely just meant to be a clarification, does it come across as passive aggressive? 😆

Reply:
a tiny bit in my opinion, dont think its a problem though haha